modules/10-volumes.sh

```diff
@@ -28,10 +28,13 @@ sed -i -e \
     /etc/fstab
 
 
-echo Scaning SCSI bus to look for new devices
-for b in /sys/class/scsi_host/*/scan; do
-    echo '- - -' > "${b}"
-done
+if [ "$(ls -A /sys/class/scsi_host)" ];
```

Comment:
set -e
is set, so your check will fail the script if that directory is missing
Merge state

Reply:
Tested it with the if statement and it bootstrapped successfully, this is a specific use case here and I don't want to deploy it on yet another branch, without it yes, the set -e causes it to fail, which I want to happen in all other cases here